src/main/java/com/asusoftware/transporter/controller/CompanyController.java

```diff
@@ -23,4 +22,14 @@
   public UUID create(@Valid @RequestBody CreateCompanyDto createCompanyDto) {
     return companyService.create(createCompanyDto);
   }
+
+  @PutMapping(path = "/{id}")
+  public void update(@PathVariable("id") UUID companyId, @Valid @RequestBody UpdateCompanyDto updateCompanyDto) {
+    companyService.update(companyId, updateCompanyDto);
+  }
+
+  @PutMapping(path = "/remove/employee")
```

Comment:
una vorbim alta facem... parca am zis ca sa folosesti verbe in path nu se practica in REST :))
de ce nu faci un DELETE pe /api/v1/employees/id?